components/transactions/TransactionRejectButton.js

```diff
@@ -77,9 +77,12 @@ const TransactionRejectButton = props => {
                 <MessageBox type="warning" mx={2}>
                   <FormattedMessage
                     id="transaction.reject.info"
-                    defaultMessage="Please only use this option if you do not wish for this contributor to be a part of your Collective. This will refund their transaction, remove them from your Collective, and display the contribution as 'rejected' in your ledger.{linebreak}{linebreak}If you are only trying to refund a mistaken transaction, please use the 'Refund' button instead."
+                    defaultMessage="Please only use this option if you do not wish for this contributor to be a part of your Collective. This will refund their transaction, remove them from your Collective, and display the contribution as 'rejected' in your ledger.{linebreak}{linebreak}{refund}"
                     values={{
                       linebreak: <br />,
+                      refund: props.canRefund
+                        ? "If you are only trying to refund a mistaken transaction, please use the 'Refund' button instead."
+                        : 'Please only use this option if you do not wish for this contributor to be a part of your Collective. This will remove them from your Collective.',
```

Comment:
This string won't be translated, resulting in mixed languages. Also, there's no need to put the line breaks and this inside the message itself, you can do it in react (it will simplify the string for translators):
```jsx
<FormattedMessage ... />
<br/>
<br/>
{props.canRefund ? <FormattedMessage ... /> : <FormattedMessage ... />}
```

Reply:
One day I will understand all the i18n best practices - I feel like I run into that the most 😅 🤦‍♀️

Reply:
Honestly it's not an obvious topic, it's easy to get things wrong and unless you actually try to translate the strings on Crowdin you don't necessary realize why some of these practices are helpful. But you do everything right on most of your recent PRs, so I think you already learnt the majority of it :wink: